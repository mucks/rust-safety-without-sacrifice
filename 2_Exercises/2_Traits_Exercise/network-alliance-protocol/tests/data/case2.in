# Alliance describe
STATE Prospera 500
STATE Zuitzerland 700
CITIZEN Prospera 10 Pax Builder
CITIZEN Prospera 11 Pia Researcher
CITIZEN Zuitzerland 20 Zoe Validator
CITIZEN Zuitzerland 21 Zed Builder
CITIZEN Zuitzerland 22 Zara Governor
ALLIANCE_NEW BlueNet
ALLIANCE_ADD BlueNet Prospera
ALLIANCE_ADD BlueNet Zuitzerland
ALLIANCE_DESCRIBE BlueNet
