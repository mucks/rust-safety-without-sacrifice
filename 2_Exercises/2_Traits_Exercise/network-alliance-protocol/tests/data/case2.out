ALLIANCE BlueNet: members=2 total_pop=5 avg_gdp=600
